models/user.js

```diff
@@ -101,6 +101,11 @@ var UserSchema = new Schema({
         type: Boolean,
         required: false,
         default: false
+    },
+    deletionDate: {
+        type: Date,
+        required: false,
+        defaut: null
```

Comment:
`defaut` typo

Reply:
Also, wouldn't it default to `null` anyways, or at least `undefined`?

Reply:
Yeah, this is redundant